### PR TITLE
Changes to guard accesses

### PR DESCRIPTION
**Pull request by Adrian16199**
## About The Pull Request
Olderly getting morgue secure mostly because most morgues have even the first door to be secure and if a doctor gets a secured one on extra access then... bit odd that olderly doesnt have it at all. Psychology because i mean.... part of medical, isnt it not? I think.

The cargo guard gets bitrunner access while destroyin the QM one because.... well, weird that they get qm access but cant get into bitrunner den.

Bouncer gets janitorial one because.... service... no? Not sure if to add chapel aswell... only if told to.

Not sure if missed any more which... well, would make sense.

## Why It's Good For The Game
I mean, its weird that some accesses are just *not there* or the access is just not correct like having QM access, just odd really.

## Proof Of Testing
Yes

## Changelog

:cl:
add: Added additional access to olderly and service guard for their respective departments.
fix: Fixes cargo guards having QM access and not bitrunner access.
:cl:

### CODE REVIEW

File: modular_skyrat/modules/goofsec/code/department_guards.dm [hunk 1]
```diff
@@ -328,25 +328,29 @@
...
 		ACCESS_WEAPONS,
+		ACCESS_PSYCHOLOGY,
 	)
 	minimal_access = list(
 		ACCESS_BRIG_ENTRANCE,
 		ACCESS_MECH_MEDICAL,
 		ACCESS_MEDICAL,
 		ACCESS_MINERAL_STOREROOM,
 		ACCESS_MORGUE,
+		ACCESS_MORGUE_SECURE
```
Comment (Majkl-J) [inline modular_skyrat/modules/goofsec/code/department_guards.dm:346]:
```suggestion
		ACCESS_MORGUE_SECURE,
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Take these upstream to skyrat

**Comment by Adrian16199** [comment]
> Take these upstream to skyrat

uuuuuuuuuuuuuuuuuuuuh..
If i can find the repository, fair.

**Comment by Majkl-J** [comment]
> uuuuuuuuuuuuuuuuuuuuh.. If i can find the repository, fair.

https://github.com/Skyrat-SS13/Skyrat-tg/pulls

**Comment by Majkl-J** [comment]
Should be taken upstream

**Pull request closed without merging**